docs/versioned_sidebars/version-zenith-sidebars.json

```diff
@@ -192,15 +196,11 @@
                 },
                 {
                   "type": "doc",
-                  "id": "developer-guide/python/custom-types"
-                },
-                {
-                  "type": "doc",
-                  "id": "developer-guide/python/visibility"
+                  "id": "developer-guide/python/ide-integration"
```

Comment:
Breaking into fundamentals and further reading is one nesting too much inside the language docs. Especially since sdks have duplicated trees, they have the same tone.

Reply:
Done